### DISCUSSION TITLE
Stream Deck control plug in

### DISCUSSION THREAD

**Discussion by nicksolyom** [dannagle/PacketSender#286] [Q&A]
Hello, 
I'd love to assign my packet sending panels to buttons on my Elgato Stream Deck. Is this possible? I could use hot keys if that exists. I just haven't been able to find the solution myself yet.
Thank all!

**Comment by dannagle** [comment]
Hello, I created #287 to support this. 

There is currently no option to access the panel button script outside the generated GUI. Setting up a series of CLI calls to packet sender is possible if you are comfortable with scripting.
